src/ucp/wireup/address.c

```diff
@@ -404,6 +404,7 @@ static int ucp_address_pack_iface_attr(ucp_worker_h worker, void *ptr,
 
     /* Keep only the bits defined by UCP_ADDRESS_IFACE_EVENT_FLAGS
      * to shrink address. */
+    ucs_assert(ucs_popcount(UCP_ADDRESS_IFACE_FLAGS) <= 22);
```

Comment:
i think it should be ucs_popcount(UCP_ADDRESS_IFACE_FLAGS) + ucs_popcount(UCP_ADDRESS_IFACE_EVENT_FLAGS)

Reply:
yes, right
fixed